control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -101,6 +101,21 @@ var (
 		// From that point the HCCO will use the OperatorHub object in the HostedCluster as a source of truth.
 		"0000_03_marketplace-operator_02_operatorhub.cr.yaml",
 	}
+
+	defaultCVOResourcesToRemove = []client.Object{
```

Comment:
feel free to suggest another variable name

Reply:
defaultResourcesToRemove